docs/blog/2019-04-18-component-shadowing/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Component Shadowing
+date: 2019-04-18
+author: John Otander
+excerpt: "Gatsby Themes introduce a concept called Component Shadowing. This feature allows
+users to override a component in order to customize its rendering."
+tags:
+  - themes
+---
+
+Gatsby Themes introduce a concept called Component Shadowing. This feature allows
+users to override a component in order to customize its rendering. Component Shadowing
+is a powerful way to make one off changes to a theme without having to fully eject.
+
+It's pretty common to end up with a theme where you're happy with 99% of the details
+but want to customize the color palette and perhaps the author bio. This is where Component
+Shadowing is an effective way to make changes quickly and intuitively.
+
+## Shadowing Example
+
+If you've installed `gatsby-theme-blog` you'll notice that it renders a
+[Bio component](https://github.com/gatsbyjs/gatsby/blob/666a9bc3c8d91be8a3118b1128340a06e895735e/themes/gatsby-theme-blog/src/components/bio.js)
```

Comment:
Is there a more stable version of this link?

Reply:
This link will always point to a version of the code that reflects the blog post, did you mean something else by stable?

Reply:
ah, i suppose that makes sense..since it is tied to a commit.

Reply:
I'll plan on updating this post as well once the official themes stabilize since there still can be some fluctuation.